node/service/src/tests.rs

```diff
@@ -168,31 +165,23 @@ impl TestChainStorage {
 			if minimum_block_number >= block.number {
 				break
 			}
-			descriptions.push(BlockDescription {
-				session: 1 as _, // dummy, not checked
-				block_hash,
-				candidates: vec![], // not relevant for any test cases
-			});
-			if !self.approved_blocks.contains(&block_hash) {
+			if self.approved_blocks.contains(&block_hash) {
```

Comment:
This logic now matches the `fn handle_approved_ancestors(...)` function in approval-voting